src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.html

```diff
@@ -155,6 +164,16 @@
 <div *ngIf="exercise.type === ExerciseType.PROGRAMMING">
     <!-- ACTION START -->
     <div class="btn-group">
+        <button
+            class="view-team"
+            jhi-exercise-action-button
+            buttonIcon="users"
+            [buttonLabel]="'artemisApp.exerciseActions.viewTeam' | translate"
+            *ngIf="exercise.teamMode && participationStatusWrapper() !== ParticipationStatus.NO_TEAM_ASSIGNED"
+            [smallButton]="smallButtons"
+            [hideLabelMobile]="true"
+            [routerLink]="['/course-management', courseId, 'exercises', exercise.id, 'teams', assignedTeamId]"
```

Comment:
Normal users should not be able to navigate to `course-management` URLs.
Can we offer the same page also on a different based on `courses/:courseId/exercises/...`

Reply:
See comment above.